soh/src/overlays/actors/ovl_En_Cow/z_en_cow.c

```diff
@@ -209,12 +216,115 @@ void func_809DF730(EnCow* this, GlobalContext* globalCtx) {
     }
 }
 
+struct CowInfo {
+    int cowId;
+    RandomizerCheck randomizerCheck;
+};
+
+struct CowInfo EnCow_GetCowInfo(EnCow* this, GlobalContext* globalCtx) {
+    struct CowInfo cowInfo;
+
+    switch (globalCtx->sceneNum) {
+        case SCENE_SOUKO: // Lon Lon Tower
+            if (this->actor.world.pos.x == -229 && this->actor.world.pos.z == 157) {
+                cowInfo.cowId = 0;
+                cowInfo.randomizerCheck = RC_LLR_TOWER_LEFT_COW;
+            } else {
```

Comment:
it might be nice to do an `else if` here and check the position of the other cow too

i don't expect it to be an issue as written, but i can imagine a scenario where:
1. we update the position of the cow from line 229
2. we forget to update the values in here
3. both cows end up returning the values from the else
4. we don't notice until someone is confused when the items the cows are giving don't match their spoilerfile

Reply:
@garrettjoecox thoughts on this one?

Reply:
Pushed!